src/app/api/cron/notifications/attendance/route.ts

```diff
@@ -0,0 +1,39 @@
+import { Tables } from "@/lib/db";
```

Comment:
Let's improve the file name. Should show the intent.

Reply:
fixed